emulatorlib/src/cpu.cpp

```
@@ -0,0 +1,67 @@
+    m_registers.hlRegister = 0x014D;
+    m_registers.wzRegister = 0x0000;
+}
+
+void Cpu::Step()
+{
+    m_handler.m_handle.resume();
+
+    if (m_handler.HasException())
+    {
+        std::rethrow_exception(m_handler.Exception());
+    }
```

Cppcheck (2.21.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)